clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -660,6 +662,26 @@ class P4RuntimeClient:
             pass
         return None
 
+    def get_packet_in(self, timeout=3):
+        msg = self.get_stream_packet("packet", timeout)
+        if msg is not None:
+            return msg.packet
+
+    def send_packet_out(self, payload):
+        packet_out = p4runtime_pb2.PacketOut()
+        packet_out.payload = payload
+        packet_out_req = p4runtime_pb2.StreamMessageRequest()
+        packet_out_req.packet.CopyFrom(packet_out)
+        self.stream_out_q.put(packet_out_req)
+
+    # Continously poll for pkts from the server
```

Comment:
Typo in "Continuously"

Reply:
Also, in comments better to say "packet" instead of "pkt"

Reply:
Done.